Speed up the landing page

baylessj opened the pull request:
Moving away from Bootstrap and the old template to a simple CSS file and no JS. Kept the home page _mostly_ the same, but added a couple of new items too.

## Lighthouse scores (on localhost)
- Mobile: 100
- Desktop: 99

## TODO

- [ ] Find a nice candid photo of PROS people doing PROS things, or remove the picture placeholder
- [ ] Get some better testimonials

## Screenshots:

Desktop
![pros-desktop](https://user-images.githubusercontent.com/16109996/101296174-00cc4d80-37f0-11eb-8f24-f8e9ce74684c.png)

Mobile
![pros-mobile](https://user-images.githubusercontent.com/16109996/101296177-05910180-37f0-11eb-999c-4c758274f107.png)

baylessj commented:
Possible testimonials:

https://www.vexforum.com/t/pros-3-1-0-release/49127/8
https://www.vexforum.com/t/pros-vs-other-programming-software/84765/2

WillXuCodes commented:
https://www.vexforum.com/t/the-pros-of-pros/59425/14?u=ranchsauce

Another testimonial (though do note it's from theo lol)

WillXuCodes commented:
By the way, I feel like we should notify those who we are using testimonials of in some capacity or another that we are using their quotes on our webpage.

baylessj commented:
I like your post from Theo best, let's go with that one. I can reach out to him and Oscar on discord to get their permission to use the testimonials.

baylessj commented:
Both Theo and Oscar are cool with the quotes, I'll update the PR with them in a bit.

baylessj commented:
![20170409_201526](https://user-images.githubusercontent.com/16109996/102697563-9e6a4880-4204-11eb-841c-6e6c3c0dc450.jpg)
![20171117_224502](https://user-images.githubusercontent.com/16109996/102697564-9f02df00-4204-11eb-91bd-2b610240c912.jpg)

So here are the two best photo candidates I could find for the PROS photo... They're all from 2017 though and not great candidates hahaha

It seems like we're not getting much from the request for photos in our discord so it might be time to strike that section from the draft

WillXuCodes commented:
Honestly, the first picture would probably be the better of the two candidates. However, another good alternative to this is to put the picture of Purdue that we already have in that spot.

baylessj commented:
I tried using the Purdue picture but it seemed a bit empty, was best off as a background image. I pulled the header image from this article about the tournament (https://www.cs.purdue.edu/news/articles/2019/SIGBots-2019.html), thoughts?

![Uploading Screenshot from 2020-12-20 20-20-08.png…]()

WillXuCodes commented:
Oh yeah that looks better for sure, good with me :+1:

baylessj commented:
@kunwarsahni01 @HotelCalifornia This PR should be ready to go so long as it's good with you, can you add your reviews whenever you get a chance?

HotelCalifornia commented:
Any chance we could re-record the gif in the background with the latest API? if not it's whatever

HotelCalifornia commented:
also maybe as a side project,  set up auto-deploy  forCI build artifacts from PRs to a staging URL so the results can be verified online (:

baylessj commented:
> Any chance we could re-record the gif in the background with the latest API? if not it's whatever

It would be possible but I don't have a Windows machine up and running at the moment, would need someone else to update the GIF with the PROS editor. We could open a separate issue for that here and get around to it eventually maybe?

> also maybe as a side project, set up auto-deploy forCI build artifacts from PRs to a staging URL so the results can be verified online (:

I would love to see that happen, not really sure what all we have in the way of a staging domain though. My first thought was to use github pages or something similar, could we do this with the existing azure-pipelines setup though?

HotelCalifornia commented:
> I would love to see that happen, not really sure what all we have in the way of a staging domain though. My first thought was to use github pages or something similar, could we do this with the existing azure-pipelines setup though?

my initial thought was to set up a branch on this repo for gh-pages to use and then have the pipeline commit the artifacts there under a folder keyed to the PR so you'd be able to get to the build for a PR at https://purduesigbots.github.io/pros-docs/204/ or something. i'm sure there's a way to do this in azure, but failing all else i have some sample github workflows we might be able to adapt

baylessj commented:
Opened #214 for this, can take a look at that sometime this week

baylessj commented:
Kunwar's suggestions from Discord:

- [ ] Fix mobile Chrome disaster
- [ ] Switch to the high res main logo
- [ ] Keep Hero subtitle text white on light theme
- [ ] Fix the left shift that of the hero content when navigating forwards/backwards
- [ ] Add a hamburger menu to the mobile version of the site

baylessj commented:
I merged master in, updated a couple of things (like mentioning VSCode instead of Atom), and updated the screenshots in the PR description.

For some added context on why I'd really like to merge this in, here's how the current webpage renders (after some time) on my machine:

![Screenshot 2022-06-06 at 11-43-07 PROS](https://user-images.githubusercontent.com/16109996/172216382-3cf773a4-6e6f-4134-95d0-7730277e0274.png)
![Screenshot 2022-06-06 at 11-44-01 PROS](https://user-images.githubusercontent.com/16109996/172216393-c52849bf-2858-47df-bdd8-6db36bb2700b.png)

kunwarsahni01 commented:
Testing this locally on a mac and looks like the download and getting started buttons link to the same page?

WillXuCodes reviewed (approved):
@kunwarsahni01 Our front page needs a bit of a touchup, let's get this done.
